config/secrets.yml

```diff
@@ -19,6 +19,7 @@
 
 development:
   secret_key_base: 33301bee9bb9a19ef446b8b0f2a3bc94b7c2a3c00869839347e485e6196a9367f20fc3d3e083c57761bd3126e45f5ab439839c1fda9dbee955f3948064729015
+  devise_mailer: <%= ENV.fetch('MAIL_SENDER', 'some_email@wolox.com.ar') %>
 
 test:
```

Comment:
We usually use the same keys for develeopment and test, so this is the only case where we use a merge. If you want to avoid repeating key/values, you can do
```yaml
development: &dev
  ...

test:
  <<: *dev
  ...
```

Reply:
Which is very similar to what you did before